prometheus.lua

```diff
@@ -261,6 +261,39 @@ local function construct_bucket_format(buckets)
   return "%0" .. (max_order + max_precision + 1) .. "." .. max_precision .. "f"
 end
 
+-- Format bucket format when expose metrics.
+--
+-- This receives a key, remove leading and trailing zeros and return the cleaner
+-- number, in order to make the output more clear, without effect on the increasing
+-- order.
+--
+-- Args:
+--   key: the metric key
+--
+-- Returns:
+--   (string) the formatted key
+local function format_bucket_when_expose(key)
+  local all_le_string = string.match(key, '(le=".*")')
```

Comment:
This will probably work correctly only if `le` is the last label. Otherwise, it'll match too much. E.g.: for `key='my_metric{le="0.10",z="x"}'` it will return `'le="0.10",z="x"'`. 

Also, it might match incorrectly if there were a label ending with 'le', e.g.: `my_metric{stale="0",le="0.01"}`.

I think this function deserves more thorough unit tests...

Reply:
Actually `le` would always be the last label in the current implementation, see https://github.com/knyar/nginx-lua-prometheus/blob/499de49cab0217c855dd9be12c3c1596cc2e9827/prometheus.lua#L323-L333 so I think we don't need to worry about that.

But the later one is what I missed before. Thanks for remind!

I tried to get the last `le=`, but when talking about sum/count, there would be no label `le` but the others say `stale`, so get the last one is not appropriate. Right now I fix this problem by filter both `,le="` and `{le="`. Do you have some other suggestions?

And new unit tests added for 1. label "stale" 2. no labels.

Reply:
Oh, I didn't check the implementation details. So it is probably ok to do it this way, but this assumption should be checked in unit tests (so it doesn't break in future, if someone changes the ordering of labels). 

Regarding the checking: You can match both variants at once:
```lua
local part1, bucket, part2 = key:match('([,{]le=")(.*)(")')
```

Reply:
Yes I've added unit tests for these scenarios, thanks for remind!

And thanks for the regex suggestions🆒